apple/bundling/plist_support.bzl

```diff
@@ -33,7 +33,14 @@ def _extract_provisioning_plist_command(ctx, provisioning_profile):
     # plist without a signature.
     return "cat " + bash_quote(provisioning_profile.path)
   else:
-    extract_plist_cmd = ("security cms -D -i " +
```

Comment:
@sergiocampama - This seems to undo the PR that merged yesterday for High Sierra.

Reply:
Yup, we didn't notice at first (or tools may be lacking in this aspect) but I cherrypicked your change back at the top of master